src/promptflow-tools/promptflow/tools/yamls/open_source_llm.yaml

```diff
@@ -6,20 +6,39 @@ promptflow.tools.open_source_llm.OpenSourceLLM.call:
   class_name: OpenSourceLLM
   function: call
   inputs:
+    endpoint_name:
+      default: "-- please enter an endpoint name --"
```

Comment:
@chjinche is there a way I can set the length of this text box?

Reply:
add @naiyunzhang @daixianfeng to take a look.

Reply:
Offline synced, unfortunately, seems no contract way to support text box size adjustment. We will take this ask into consideration in custom tool cap design.